src/apiMeetup.js

```diff
@@ -0,0 +1,3 @@
+export const getMeetupEvents = () => {
+  return fetch(`https://api.meetup.com/Montpellier-PHP-Meetup/events?status=past`)
```

Comment:
I think you are waiting from meetup to approve your app. Let's move on this to fix it later on

Reply:
This will not work anymore, Meetup has moved to [OAuth only](https://help.meetup.com/hc/en-us/articles/360028901812-Using-Meetup-s-API) 
We can remove all the flow for new to rebiuld it later